azure.yaml

```diff
@@ -8,16 +8,30 @@ services:
     project: ./app/backend
     language: py
     host: appservice
+  scripts:
+    project: ./scripts
+    language: py
+    host: ""
 hooks:
     postprovision:
-      shell: pwsh
-      run: ./scripts/prepdocs.ps1
-      interactive: true
-      continueOnError: false
+      windows:
+        shell: pwsh
+        run: ./scripts/prepdocs.ps1
+        interactive: true
+        continueOnError: false
+      posix:
+        shell: sh
+        run: scripts/scripts_env/bin/python scripts/prepdocs.py 'data/*' --storageaccount $AZURE_STORAGE_ACCOUNT --container $AZURE_STORAGE_CONTAINER --searchservice $AZURE_SEARCH_SERVICE --index $AZURE_SEARCH_INDEX -v
```

Comment:
I don't think we can guarantee the script_env venv.  Why not have the venv create in the .sh file?

Reply:
I assumed that we want to make use of `azd restore` - isn't that why it creates a virtual environment per "service" in the first place? And since it is `azd restore` that creates the virtual environment, I'd think that there is some way to plumb the environment name through here....